internal/db/update/schema.go

```diff
@@ -156,11 +163,19 @@ func (s *SchemaUpdate) Dump(db *sql.DB) (string, error) {
 	}
 
 	// Add a statement for inserting the current schema version row.
-	statements = append(
-		statements,
-		fmt.Sprintf(`
-INSERT INTO schemas (version, updated_at) VALUES (%d, strftime("%%s"))
-`, len(s.updates)))
+	for _, versionType := range []int{0, 1} {
+		updates := s.internalUpdates
+		if versionType == 1 {
```

Comment:
yeah we need to use constants - otherwise its going to get confusing.

Reply:
question: why have two variables s.internalUpdates and s.externalUpdates, rather than s.updates being a map[int][]schema.Update?

Also we should get into the habit of using "thing{Action/Type}" naming like we do elsewhere, so if we stick with 2 vars, they should be `updatesInternal` and `updatesExternal`.

Reply:
Could we introduce a data type for the schema type too so it becomes: `map[schemaType]schema.Update`?

Reply:
I've made the schema update lists into slices now, but this may have adverse effects on any project that didn't structure their external schema update map quite like our internal one with ordered indexes.

Reply:
The micro-projects are all fine on this front.